ethereum/beaconchain/src/main/java/tech/pegasys/artemis/Constants.java

```diff
@@ -33,7 +33,7 @@
   public static final int MAX_CASPER_VOTES                               = (int) Math.pow(2, 10); // 1,024 votes
   public static final int LATEST_BLOCK_ROOTS_LENGTH                      = (int) Math.pow(2, 13); // 8,192 block roots
   public static final int LATEST_RANDAO_MIXES_LENGTH                     = (int) Math.pow(2, 13); // 8,192 randao mixes
-  public static final UInt384[] EMPTY_SIGNATURE                          = new UInt384[]{UInt384.valueOf(0), UInt384.valueOf(0)};
+  public static final Bytes48[] EMPTY_SIGNATURE                          = new Bytes48[]{Bytes48.ZERO, Bytes48.ZERO};
```

Comment:
This is fine for now, but we will need to decide if we care about just the X coord of the BLS sig (called compressed form) or both the X and Y.

Reply:
good catch on the type.  about the signatures, i opted to follow what the spec called out.  we can revisit the decision on a case by case basis if we decide to go a diff direction

Reply:
Sounds good.